Command/ExtractTranslationCommand.php

```diff
@@ -180,7 +180,7 @@ private function updateWithInput(InputInterface $input, ConfigBuilder $builder)
             $builder->setOutputFormat($outputFormat);
         }
 
-        if ($input->hasParameterOption('intl-icu')) {
+        if ($input->hasParameterOption('--intl-icu')) {
             $builder->setUseIcuMessageFormat(true);
         }
```

Comment:
To use `getOption` is probably better:

```suggestion
        $builder->setUseIcuMessageFormat($input->getOption('intl-icu'));
```

Reply:
Using `getOption('intl-icu')` would override the config every time (when a config option is set), which is not expected?